addon/serializers/children.js

```diff
@@ -0,0 +1,12 @@
+import OsfSerializer from './osf-serializer';
+
+export default OsfSerializer.extend({
+    serialize: function(snapshot, options) {
```

Comment:
From http://emberjs.com/api/data/classes/DS.RESTSerializer.html#method_payloadKeyFromModelName
> You can use payloadKeyFromModelName to override the root key for an outgoing request.

I don't want to override the root key, I want to change the type from 'children' to 'nodes', because we don't support the 'children' type in the API.